tests/test_api.py

```diff
@@ -333,15 +332,6 @@ def test_metadata_timestamp(self) -> None:
         timestamp.signed.bump_expiration(timedelta(days=365))
         self.assertEqual(timestamp.signed.expires, datetime(2031, 1, 2, 0, 0))
 
-        # Test whether dateutil.relativedelta works, this provides a much
-        # easier to use interface for callers
-        delta = relativedelta(days=1)
-        timestamp.signed.bump_expiration(delta)
-        self.assertEqual(timestamp.signed.expires, datetime(2031, 1, 3, 0, 0))
-        delta = relativedelta(years=5)
-        timestamp.signed.bump_expiration(delta)
-        self.assertEqual(timestamp.signed.expires, datetime(2036, 1, 3, 0, 0))
-
```

Comment:
This test code is removed as if we don't make `timedelta` argument of `Signed` a `Union[timedelta, relativedelta]` then `mypy` complains.